ppafm/common.py

```diff
@@ -104,7 +111,7 @@ class CLIParser(ArgumentParser):
         "Rcore": {
             "action": "store",
             "type": float,
-            "default": params["Rcore"],
+            "default": 0.7,
```

Comment:
Shouldn't this be `PpafmParameters.Rcore` so that it always uses the same default as above even if it's changed?

Reply:
done in https://github.com/Probe-Particle/ppafm/pull/275/commits/2e825103ecb25625fe6cd95fa3e614ccf274c855